x-pack/platform/plugins/shared/alerting_v2/server/lib/director/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const ALERT_TRANSITIONS_INDEX = '.alerts-transitions';
+export const ALERT_EVENTS_INDEX = '.alerts-events';
```

Comment:
Asked the same question in Dima's PR => https://github.com/elastic/kibana/pull/247472#discussion_r2653635539

Should we use `_` or `-`? and should it be `.alert_` / `.alert-`.

```suggestion
export const ALERT_TRANSITIONS_INDEX = '.alert_transitions';
export const ALERT_EVENTS_INDEX = '.alert_events';
```